spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableModule.java

```diff
@@ -47,8 +47,10 @@
 @Configuration
 @MessageEndpoint
 @Import({RedisServiceConfiguration.class, RabbitServiceConfiguration.class,
-		ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, LifecycleConfiguration.class,
-		AggregateBuilderConfiguration.class, EnableModuleConfiguration.class})
+	ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, LifecycleConfiguration.class,
+	AggregateBuilderConfiguration.class, EnableModuleConfiguration.class})
 public @interface EnableModule {
 
+	Class<?>[] value() default {};
```

Comment:
The old way - pick up `@Bean` definitions and sort them by name in to input and output.